### PR TITLE
Feat/footer

### PR DESCRIPTION
**Pull request by itnoj15**
- Populated publication data
스크린샷 2024-05-01 오후 3 25 05

- Added a footer
스크린샷 2024-05-01 오후 3 24 16

### CODE REVIEW

File: src/data/publications.ts [hunk 1]
```diff
@@ -35,88 +35,2253 @@ export interface Publication {
...
+  },
+  {
+    title: 'FABRIC: Automated Scoring and Feedback Generation for Essays',
+    authors: [
+      'Jieun Han',
+      'Haneul Yoo',
+      'Junho Myung',
+      'Minsun Kim',
+      'Hyunseung Lim',
+      'Yoonsu Kim',
```
Comment (jhw123) [inline src/data/publications.ts:73]: Perhaps, we should have worked on the `members` data first...

Comment (jhw123) [reply src/data/publications.ts:73]: Plus, I think it is better to restrict the type of authors to either Members and string, not both. If we go with Members, we can keep the single source of truth policy, but we need to change the current Member class to handle external members. If we go with string, it is easier but the code will be susceptible to typos...

**Review by jhw123 — CHANGES_REQUESTED** [review]

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+`
+
+const FooterText1 = styled.p`
+  ${FontVariant.body_lg}
+  color: ${Color.white};
+  margin: 0;
+`
+
+const FooterText2 = styled.p`
+  font-size: ${FontSize.body_lg};
```
[the remark this answers is not in the document]
Comment (itnoj15) [reply src/components/Footer.tsx:29]: I think so.. Then, I will change it after discussing the design with other members!

File: src/data/publications.ts (hunk 1 above)
Comment (itnoj15) [reply src/data/publications.ts:73]: Yes, I agree! I think we need to discuss this problem as well..

File: src/components/Footer.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (alextio) [reply src/components/Footer.tsx:30]: There were cases where i had to use `FontSize` and `FontWeight` separately. 
Pragmatically, I think it is best to allow the use of `FontSize` and `FontWeight` when needed.

File: src/data/publications.ts (hunk 1 above)
Comment (alextio) [reply src/data/publications.ts:73]: I wonder if there is an API somewhere that can automatically populate these fields

**Review by jhw123 — APPROVED** [review]

**Pull request merged**